src/rewrite_generic.jl

```diff
@@ -69,7 +69,7 @@ function _rewrite_generic(stack::Expr, expr::Expr)
         # If the first argument is a splat.
         return esc(expr), false
     elseif _is_generator(expr) || _is_flatten(expr) || _is_parameters(expr)
-        if !(expr.args[1] in (:sum, :Σ, :∑))
+        if !_is_sum(expr.args[1])
```

Comment:
I think I wrote this explicitly because `_is_sum` was in `rewrite.jl`, and because it was a bit opaque exactly what we supported here.